manager/defaultmanager/copy.go

```diff
@@ -0,0 +1,165 @@
+package defaultmanager
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
+	load "github.com/uor-framework/uor-client-go/config"
+	"github.com/uor-framework/uor-client-go/model"
+	"github.com/uor-framework/uor-client-go/ocimanifest"
+	"github.com/uor-framework/uor-client-go/registryclient"
+	"github.com/uor-framework/uor-client-go/util/workspace"
+	"oras.land/oras-go/v2/content/memory"
+)
+
+func (d DefaultManager) Update(ctx context.Context, space workspace.Workspace, src string, dest string, add bool, remove bool, client registryclient.Client) (string, error) {
+
+	// Pull the dataset config from the target collection
+	_, manBytes, err := client.GetManifest(ctx, src)
```

Comment:
I wonder if this type of action could benefit from a dry run or if it could just construct the workspace and pass it to the top-level Build method. Just so the user can validate the workspace before any references are overwritten.